components/boards/Playboard.js

```diff
@@ -147,6 +100,7 @@ function Playboard({ size, ...props }) {
                 }}
                 name={`hexagon_${index}`}
                 value={value}
+                aria-label={`hexagon_${index}_x_${rowIndex}_y_${columnIndex}`}
```

Comment:
Is this a label meant to be read by a human ? I suggest:
```suggestion
                aria-label={`Hexagon at row ${rowIndex} and column ${columnIndex}`}
```

Reply:
Fixed, TY